CharlieBackend.Api/Controllers/EventsController.cs

```diff
@@ -30,6 +31,25 @@ public EventsController(IEventsService eventsService)
             _eventsService = eventsService;
         }
 
+        /// <summary>
+        /// Add single event
+        /// </summary>
+        /// <remarks>
+        /// With this endpoint you can add a single event.
+        /// </remarks>
+        /// <response code="200">Successful add of schedule</response>
+        /// <response code="HTTP: 400, API: 0">Can not create event due to wrong request data</response>
+        /// <response code="HTTP: 404, API: 3">Can not create event due to missing request data</response>
+        [SwaggerResponse(200, type: typeof(CreateSingleEventDTO))]
+        [Authorize(Roles = "Secretary, Admin")]
+        [HttpPost]
+        public async Task<ActionResult<CreateSingleEventDTO>> AddSingleEvent([FromBody] SingleEventRequestDto request)
```

Comment:
add async to name

Reply:
I think, we don't need AddSingleEventAsync, AddSingleEvent looks better.

Reply:
@BohdanZaliubovskyi base on Microsoft conventions we should add Async suffix to any method that have async keyword https://docs.microsoft.com/en-us/dotnet/csharp/async
> Add "Async" as the suffix of every async method name you write.

Reply:
It's because of current naming in project(without async in names of methods).
But, good practice means that we don't see async word in url, so we should change all names.

Reply:
> It's because of current naming in project(without async in names of methods). But, good practice means that we don't see async word in url, so we should change all names.
We can set up routing to not see async word in URL, meantime our async methods will follow Microsoft naming conventions